### DISCUSSION TITLE
Grafana settings for disabling dashboard import from UI

### DISCUSSION THREAD

**Discussion by vamsi-gk** [grafana/grafana#45106] [General]
Hi, We want to disable loading new dashboards using url from Grafana UI. is there is any setting available for this?
Vamsi

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
